doc/dependency_decisions.yml

```
@@ -2,7 +2,7 @@
 - - :restrict
   - unknown
   - &1
-    :who:
+    :who: 
```

I guess it doesn't hurt, but ... sorry I have to point it out 😛 
```suggestion
    :who:
```

Same for all other changed lines.

---

this is generated by the `license_finder` CLI :(((((

---

Hmm... that's unfortunate, but if it's generated, I am fine to live with it :D